bin/fathom-generate-events/eco-router-platoforms.ts

```diff
@@ -0,0 +1,33 @@
+import { BaseRoutablePlatform, RoutablePlatform, UniswapV2RoutablePlatform } from '@swapr/sdk'
```

Comment:
typo on the file name, should be `eco-router-platforms`

Reply:
Fixed